README.md

```diff
@@ -62,6 +62,8 @@ $ cd rocketmq-operator
 $ make deploy
 ```
 
+If you get error `rocketmq-operator/bin/controller-gen: No such file or directory`, please run `go version` to check the version of Golang, the main version should be 1.16. Then run `go mode tidy` before run `make deploy`.
```

Comment:
`mod` instead of `mode`?

Reply:
> `mod` instead of `mode`?

sorry for my typo, it should be "mod", not "mode".

Reply:
Hi @koalawangyang, could you pls also fix this typo?

Reply:
> Hi @koalawangyang, could you pls also fix this typo?

Hi @caigy , yes, of course, I have fixed the typo and re-commit it. Please check whether it's ok now. Thanks~